Zend/Optimizer/zend_func_info.c

```diff
@@ -88,81 +88,7 @@ static uint32_t zend_range_info(const zend_call_info *call_info, const zend_ssa
-	F1("pack",                         MAY_BE_STRING),
-	F1("unpack",                       MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_ANY),
-	F1("get_browser",                  MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_OBJECT | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_ANY),
-	F1("crypt",                        MAY_BE_STRING),
-	F1("getcwd",                       MAY_BE_FALSE | MAY_BE_STRING),
-	F1("readdir",                      MAY_BE_FALSE | MAY_BE_STRING),
-	F1("dir",                          MAY_BE_FALSE | MAY_BE_OBJECT),
-	F1("scandir",                      MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_STRING),
-#ifdef HAVE_GLOB
-	F1("glob",                         MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_STRING),
-#endif
-	F1("filetype",                     MAY_BE_FALSE | MAY_BE_STRING),
-	F1("stat",                         MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_FALSE | MAY_BE_ARRAY_OF_TRUE | MAY_BE_ARRAY_OF_LONG | MAY_BE_ARRAY_OF_STRING),
-	F1("lstat",                        MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_FALSE | MAY_BE_ARRAY_OF_TRUE | MAY_BE_ARRAY_OF_LONG | MAY_BE_ARRAY_OF_STRING),
-	F1("realpath_cache_get",           MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_ARRAY),
-#ifdef HAVE_SYSLOG_H
-	F0("syslog",                       MAY_BE_TRUE),
-	F0("closelog",                     MAY_BE_TRUE),
-#endif
-	F1("metaphone",                    MAY_BE_STRING),
-	F1("ob_get_flush",                 MAY_BE_FALSE | MAY_BE_STRING),
-	F1("ob_get_clean",                 MAY_BE_FALSE | MAY_BE_STRING),
-	F1("ob_get_status",                MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_LONG | MAY_BE_ARRAY_OF_STRING | MAY_BE_ARRAY_OF_ARRAY),
-	F1("ob_list_handlers",             MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_STRING),
-	F0("array_walk",                   MAY_BE_TRUE),
-	F0("array_walk_recursive",         MAY_BE_TRUE),
-	F0("arsort",                       MAY_BE_TRUE),
-	F0("asort",                        MAY_BE_TRUE),
-	F0("krsort",                       MAY_BE_TRUE),
-	F0("ksort",                        MAY_BE_TRUE),
-	F0("shuffle",                      MAY_BE_TRUE),
-	F0("sort",                         MAY_BE_TRUE),
-	F0("usort",                        MAY_BE_TRUE),
-	F0("uasort",                       MAY_BE_TRUE),
-	F0("uksort",                       MAY_BE_TRUE),
-	F1("compact",                      MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_REF | MAY_BE_ARRAY_OF_ANY),
-	FN("array_fill",                   MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_ANY),
-	F1("array_fill_keys",              MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_REF | MAY_BE_ARRAY_OF_ANY),
 	FC("range",                        zend_range_info),
```

Comment:
Any way to do eliminate this from `old_func_infos`?

Reply:
Don't think so